bin/PerlDDS/Run_Test.pm

```diff
@@ -1025,6 +1052,30 @@ sub _create_process {
   return $proc;
 }
 
+sub _create_java_process {
+  my $self = shift;
+  my $main_class = shift;
+  my $params = shift;
+  my @jars = shift;
+  my $vmargs = shift;
+
+  $self->_info("TestFramework::_create_java_process creating executable="
+    . "w/ params=$params jars=@jars vmargs=$vmargs\n");
```

Comment:
Since this is actually an array reference it won't turn into anything useful (something like `ARRAY(0x564ba59fd4b8)`). Arrays of strings will become strings, but they don't have separators between the elements.

```suggestion
    . "w/ params=$params jars=" . join(':', @{$jars}) . " vmargs=$vmargs\n");
```

I put in `:` since that's how the Java classpaths are separated, but it could be comma, space, etc.